main.lua

```diff
@@ -350,7 +367,9 @@ end
 
 function defaultOnErrorHandler(method)
     return function(error)
-        display_error(("%s (Error %d, %s)"):format(error.message, error.code, method))
+        display_error(("%s (Error %s, %s)"):format(
```

Comment:
The word "Error" is unnecessary now that we have the actual name of the error. It might also make sense to put the error code before the message now that it is human readable.